### DISCUSSION TITLE
Can you provide the Reference for Graph Codes and Notes

### DISCUSSION THREAD

**Discussion by nihalahmadmulla** [VinayakJamadar/DSA#4] [Q&A] [answered]
Can you provide the Reference for Graph Codes and Notes ?

**Comment by VinayakJamadar** [comment, marked as answer]
Sure!
Below are Reference for Graph
Videos Link: https://www.youtube.com/playlist?list=PLgUwDviBIf0oE3gA41TKO2H5bHpPd7fzn
Notes Link: https://takeuforward.org/graph/striver-graph-series-top-graph-interview-questions/

I also added the above Reference in the README.md file
